blog/content/post/YYYY-MM-DD-template-post.md

```diff
@@ -22,6 +25,9 @@ categories:
 
 Your post content
 
+> post abstract in the header is required for the posts summary in the blog list
+  and must contain from 3 to 5 sentences
```

Comment:
```suggestion
  and must contain from 3 to 5 sentences, please note that abstract would be used
  for social media and because of that should be focused on keywords/hashtags
```